Extensions/ValuesOfMultipleObjects.json

```diff
@@ -0,0 +1,1796 @@
+{
+  "author": "Tristan Rhodes (https://victrisgames.itch.io/)",
+  "description": "Provides values based on picked object instances.  Useful for camera tracking, flocking behaviors, and more.\n\nIncluded expressions:\n\nCenter position (average)\n- X center position \n- Y center position\n\nCenter position (middle)\n- X center position\n- Y center position \n\nBased on center of objects:\n- Minimum X position\n- Maximum X position\n- Minimum Y position\n- Maximum Y position\n\nBased on AABB of objects:\n- Minimum X position\n- Maximum X position\n- Minimum Y position\n- Maximum Y position\n\nForce related expressions:\n- Average horizontal force (X)\n- Average vertical force (Y)",
+  "extensionNamespace": "",
+  "fullName": "Values of multiple objects",
```

Comment:
🤔 Personally, I am not very fond of that name as it implies that the extension gets any kind of value from multiple objects, where it is only position related ones (X, Y and Forces). Maybe `Positions of multiple objects` ?

Reply:
I want this helper extension to grow as use-cases are identified.   For instance, I just added an "AverageAngle" function.

Reply:
Ah in that case it's fine then